src/components/AboutPage/components/AboutDesription/AboutDescription.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './AboutDescription.scss';
+
+export const AboutDesription = () => {
+  return (
+    <section className="about-description">
+      <h2 className="about-desription-header">What we do</h2>
+      <p className="about-desription-text">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Recusandae odit
+        reprehenderit ipsam neque nemo ducimus aliquam consequatur iusto
+        voluptate fugiat temporibus rem voluptatibus amet dolorum perspiciatis,
+        quas, accusamus consectetur! Quam quis, at obcaecati, ex amet ipsum
+        numquam odio in vitae voluptatem fuga, error velit quaerat delectus
+        necessitatibus excepturi libero tenetur praesentium laboriosam voluptas
+        ad? Sapiente, perferendis suscipit expedita deserunt deleniti iusto
+        ipsum quas sit. Quos odio commodi repellat non, recusandae ipsum iusto,
+        necessitatibus delectus repellendus fugit voluptate enim cumque,
+        repudiandae harum vero veritatis blanditiis sit quia dicta? Unde, itaque
+        suscipit inventore enim id nobis nostrum maiores, autem dolorem ducimus
+        eligendi.
+      </p>
```

Comment:
Use a more appropriate description

Reply:
It seems to me, that Typography should be used here

Reply:
Typography added
Description is changed